data/accounts/store.go

```diff
@@ -46,6 +46,7 @@ func secureUser(user *User) *User {
 	return user
 }
 
+// nolint : dupl
 func (s *Store) getUser(ctx context.Context, name string) (user *User, err error) {
 	if user, err = s.rawUser(ctx, name); err != nil {
```

Comment:
```
if user, err := s.rawUser(ctx, name); err != nil {
		return nil, err
	}
```

This doesn't work since the `user` variable is only defined in the `if` scope, which is why, by convenience over consistency, I used a named return value.